src/storage/test/IndexScanTest.cpp

```diff
@@ -371,6 +373,7 @@ static cpp2::LookUpEdgeIndexResp checkLookupEdgesString(const std::string& filte
     }
     std::unique_ptr<meta::IndexManager> indexMan(im);
     auto eindex = indexMan->getEdgeIndex(spaceId, type).value();
+    sleep(FLAGS_raft_heartbeat_interval_secs);
```

Comment:
why sleep here?

Reply:
> why sleep here?

Execute the data insert immediately when the schema created just done. some times individual part have not ready yet , so using sleep waiting a heartbeat let to make partitioned folder and waiting part leader ready.

Reply:
OK